mods/ts/weapons/healweapons.yaml

```diff
@@ -1,25 +1,25 @@
-Heal:
+^HealWeapon:
 	ReloadDelay: 80
 	Range: 2c849
 	Report: healer1.aud
-	ValidTargets: Infantry
-	Projectile: Bullet
-		Speed: 1c682
+	Projectile: InstantHit
 	Warhead@1Dam: SpreadDamage
```

Comment:
Okay, but ` If it's a WPos then search for all actors with an overlapping hit shape, and pick the one that is closest (or all?)` means that it will need a victim scan radius, and it would save some work on both the warhead and lint rule if #12605 was finished first, which means this PR would likely sit idle for a while if you consider it a blocker.
Personally, I'd prefer to just add a TODO here for now, and then deal with the heal weapons as part of a dedicated `TargetDamageWarhead` PR (as they'd make for a good testcase).